testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -123,6 +123,10 @@ impl RunLoop {
         let pox_constants = burnchain.get_pox_constants();
 
         let is_miner = if self.config.node.miner {
+            // Initialize the wallet.
+            info!("Miner node: creating the wallet if it does not exist");
+            burnchain.create_wallet_if_dne();
```

Comment:
Because this is a pretty expensive operation in Bitcoin mainnet and testnet, I think we should avoid trying to do this automatically for users. Can this call be moved to the `neon_integrations` setup code? That way the new functions in the regtest controller could also be marked `cfg(test)`

Reply:
I don't know what setup code you're referring to in `neon_integrations`? For now I replicated the way `bump_blocks_processed` handles test/non-test code in `neon.rs`.

Reply:
All of the `neon_integrations` tests do some form of the following:

```rust
    let mut btcd_controller = BitcoinCoreController::new(conf.clone());
    btcd_controller
        .start_bitcoind()
        .map_err(|_e| ())
        .expect("Failed starting bitcoind");

    let mut btc_regtest_controller = BitcoinRegtestController::new(conf.clone(), None);
    let http_origin = format!("http://{}", &conf.node.rpc_bind);

    btc_regtest_controller.bootstrap_chain(201);
```

Maybe we could call `create_wallet_if_dne` from the `bootstrap_chain` implementation of the `bitcoin_regtest_controller`? That way it could also be non-public.